docs/buildah-build.1.md

```diff
@@ -697,6 +697,8 @@ The secret will be mounted in the container at the default location of `/run/sec
 To later use the secret, use the --mount flag in a `RUN` instruction within a `Containerfile`:
 
 `RUN --mount=type=secret,id=mysecret cat /run/secrets/mysecret`
+    
+Warning: Changing the contents of secret files will not trigger a rebuild of layers that use said secrets.
```

Comment:
This should be called `Note` not `Warning` since this is expected behavior :)

Reply:
Force-pushed a fix.